.github/workflows/install.yml

```diff
@@ -20,8 +20,33 @@ jobs:
         run: apt-get update && apt-get -y install build-essential cmake curl git
       - name: Install Rust
         run: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-      - name: Install Pop
+      - name: Pop Install
```

Comment:
I think it would be better to separate this into two steps, so that it is easier to parse the output of the run. Suggestion is "Install Pop" as before, without the 'pop --version' and then a separate step for "Pop Install".

See the current workflow output as an example.

Reply:
Done!